### ISSUE TITLE
Connect with phone number!

### ISSUE THREAD

**Issue by EminDemiri** [QuickBlox/q-municate-web#84]
Meanwhile, I ask pardon for the opening of another problem.

I want to know how to set up Connect with phone number!

thank you

**Comment by Vladlukhanin** [comment]
Yon need to create and configure twitter digits - https://fabric.io/kits
And when you get a key customer, set it here - https://github.com/QuickBlox/q-municate-web/blob/master/app/scripts/main.js#L137

**Comment by EminDemiri** [comment]
Could you explain better coma do to create an account?

I registered on fabric.io.
Every time I click on the dashboard reminds me of kits page.

If I click on twitter it opens a screen where the installation tells me that I have a twitter application. There'm not understanding anything you can help me? :)

**Comment by EminDemiri** [comment]
I take all the steps for creating a key bees but does not work, I always get an error message.

**Comment by EminDemiri** [comment]
I created an application on dev.twitter.com I did a test and rightly did not work as written on some comments.

I re-read the documentation and I have created an application with android studio, I installed the plugin Fabric and I followed the procedure to insert figits within the app. What did I let the app that has started on my device, immediately after the project is on my dashboard compartment Fabric.io.
I came in and changed the WEBSITE golf fabric.io to talkme.online, I have not done any other notification.

Once I've copied the saved CONSUMER KEY (KEY API) and I glued in the predetermined range of my web app.

Once I uploaded to the web when I go to open the link to login paggina I see this error:

You have attempted to access Digits by Twitter from a different site from the URL registered for.

If you are the owner of the site, check the documentation of Digits to the Web to learn more about how to record the correct URL.

▸ More details
Missing host parameter in the URL

This is the url field of the page:

https://www.digits.com/login?consumer_key=kDzUhpfeDvhuFtyg3GcErbeSs&host=http%3A%2F%2Ftalkme.online

It states that the domain name is correct as well as the consumer_key is the same present in the dashboard fabric.io

What should I do to make the login via sms digits?
